### PR TITLE
Handle kernel versions with '_' before build number

### PR DESCRIPTION
**Pull request by SandakovMM**

### CODE REVIEW

File: pleskdistup/common/tests/versiontests.py [hunk 1]
```diff
@@ -34,6 +34,9 @@ def test_kernel_start_with_prefix(self):
     def test_kernel_start_with_plus_prefix(self):
         self._check_parse("kernel-plus-3.10.0-327.36.3.el7.centos.plus.x86_64", "3.10.0-327.36.3.el7.centos.plus.x86_64")
 
+    def test_kernel_with_underline(self):
+        self._check_parse("kernel-3.14.43_1-2.x86_64", "3.14.43-1.2.x86_64")
```
Comment (kpushkaryov) [inline pleskdistup/common/tests/versiontests.py:38]: The versions are getting convoluted. I propose to test components of `KernelVersion` separately. That is, in addition to testing string representation, also test that `major`, `minor`, etc. are correct.

**Review by kpushkaryov — CHANGES_REQUESTED** [review]

File: pleskdistup/common/tests/versiontests.py (hunk 1 above)
Comment (SandakovMM) [reply pleskdistup/common/tests/versiontests.py:38]: > The versions are getting convoluted.
Yep. Unfortunately, I don't think we could do much about it.
Let's add some test here as you proposed.

File: pleskdistup/common/tests/versiontests.py [hunk 2]
```diff
@@ -34,6 +34,16 @@ def test_kernel_start_with_prefix(self):
     def test_kernel_start_with_plus_prefix(self):
         self._check_parse("kernel-plus-3.10.0-327.36.3.el7.centos.plus.x86_64", "3.10.0-327.36.3.el7.centos.plus.x86_64")
 
+    def test_kernel_with_underline(self):
+        kernel = version.KernelVersion("kernel-3.14.43_1-2.x86_64")
+        self.assertEqual(str(kernel), "3.14.43-1.2.x86_64")
+        self.assertEqual(kernel.major, "3")
```
Comment (kpushkaryov) [inline pleskdistup/common/tests/versiontests.py:40]: I suggest to expand other tests in the same way later.

**Review by kpushkaryov — APPROVED** [review]

File: pleskdistup/common/tests/versiontests.py (hunk 2 above)
Comment (kpushkaryov) [reply pleskdistup/common/tests/versiontests.py:40]: Please rebase before merge.

**Comment by SandakovMM** [comment]
@kpushkaryov Could you please approve once again. Changes from other commit leaked into the branch, so I have to remove it and re-base once again.

**Review by kpushkaryov — APPROVED** [review]

**Pull request merged**